scenarios/smart_trailer/scripts/start_trailer_applications_bluechi.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+# SPDX-License-Identifier: MIT
+
+set -e
```

Comment:
nit: I believe you could add u here as well `-eu`. -u tells bash to "Treat unset variables as an error when substituting."